script/Ciel/Day 14/14_01A_CIEL14A_2.txt

```diff
@@ -515,42 +559,54 @@ While I'm still perplexed, Senpai grabs on to my arm and forces me to run.
 [sha:ac32de7b98ee360bc8b89a8b0f9d364b8ec8ac63]{
 -- Page 48, Offset 37751.
 -- 「どうしてもなにも、聞きたいのはこっちのほうです！
-"If anything, I'm the one who should be asking questions here!
+"What do you mean 'why',
+ %{i}I'm%{/i} the one who should be asking the questions here!
 }
 [sha:7d81f7601424470834a28d71e74e5d3bdcac0863]{
 -- Page 48, Offset 37752.
 -- 急いで帰ってきてみれば公園<で時空が|が><歪むぐら|ア><いの圧力|レ>があっ<て|で>、アルクェイドがアレなんですから……！」
-I tried to come back quickly, but then the park was like that, and Arcueid was also there...!"
+// Fuck off with this ruby
+// Using the base sentence since without it the followup doesn't make sense.
+I came back as quickly as I could only to find the park filled with enough pressure to distort space-time,
+ and Arcueid looking like %{i}that%{/i}!"
 }
 [sha:24e50eab75d584df18b05a6cfe6d6eb085534d1a]{
 -- Page 49, Offset 37753.
 -- ―――世界が凍るような圧迫感は錯覚ではなく、ただの現実だったらしい。
-―――It seems that the feeling of oppression like the world had frozen over wasn't an illusion, but a reality.
+―――I guess that oppressive feeling that the world had stopped wasn't an illusion, but reality.
 }
 [sha:74519b978bffd05ecc3a7ddc4e12b5f64eea9d80]{
 -- Page 49, Offset 37754.
 -- それはともかく、その異常を前にして即座にあれだけの対処を行えるシエル先輩も負けてはいない。
-But that aside, Ciel-senpai was also impressive in handling it immediately in the face of such an abnormal situation.
+Even so, Ciel-senpai is no less impressive for immediately taking decisive action in the face of such an abnormal situation.
 }
 [sha:1caa76cc99206ef5656c9dcc9c40438dfaeeed94]{
 -- Page 50, Offset 37755.
 -- 「……すみません、部屋から出たのは謝ります！
-"...I'm sorry, I shouldn't have left the room!
+"I'm sorry, I shouldn't have left the room!
 }
 [sha:6b3ab84580e1b6e604688c78860a8cb216af9d1c]{
 -- Page 50, Offset 37756.
 -- 頭の痛みにどうしても耐えられなくて、外に出たらアルクェイドがいて、アイツ、自分のものになれとか言い出して―――」
-I couldn't handle the pain in my head, and when I went out, Arcueid was there, and she told me to become her thing―――"
+// I guess referring to 37387?
+// 方に、わたしの下僕になれと言っているのよ
+// not a great match.
+I just couldn't handle the pain in my head,
+ and when I went outside,
+ I ran into Arcueid,
+ and she told me to become hers―――"
 }
 [sha:f4148fe3d5935270339fe9dfd1f281072dc91a4d]{
 -- Page 51, Offset 37757.
 -- 「―――そうですか。アルクェイドは、ロアと遠野くんを混同しているんですね。……やはり、彼女との対決は……」
-"―――Is that right? So Arcueid has put herself between you and Roa... As I thought, a showdown against her is..."
+"―――I see.
+ So Arcueid has confused you with Roa...
+ As I feared, a showdown against her is..."
```

Comment:
Just some phrasing stuff
She also kinda interrupts herself, so...
>―――So that's it.
>Arcueid is conflating you with Roa...
 As I feared, a showdown against her is———